packages/openbridge-webcomponents/src/automation/automation-tank/automation-tank.css

```
@@ -2,63 +2,292 @@
+  height: 100%;
+  margin: 0;
+  padding: 0;
+  border: none;
+  background: transparent;
+  appearance: none;
+  display: flex;
+  flex-direction: column;
+  gap: 4px;
+  cursor: pointer;
+  /* prettier-ignore */
+  @mixin style style=flat visibleWrapperClass=.halo;
```

Stylelint (17.10.0)

[error] Expected mixin name "style style=flat visibleWrapperClass=.halo" to be kebab-case (scss/at-mixin-pattern)

(scss/at-mixin-pattern)